src/index.js

```diff
@@ -387,6 +387,19 @@ module.exports = async ({
         },
         json: true,
       }).then(R.prop("body")),
+    getAccount: async (accountId, userId) => {
```

Comment:
Could we change the userId to be the first parameter? 
We were thinking on changing the getAccounts method to only receive the user id as a parameter and request a token inside the method the same way you did it here.

Reply:
Yeah that's fine, i've updated the code, readme & examples in https://github.com/moneyhub/moneyhub-api-client/pull/2/commits/46dff18d6cb1c29fd06f7ed926bd584dcdc64243